osgtest/tests/test_450_xrootd.py

```diff
@@ -3,6 +3,7 @@
 import shutil
 import tempfile
 import pwd
+from os import stat
```

Comment:
I don't think this is used anywhere since you're using `core.check_file_ownership` instead
```suggestion
```